[CQA] Fix feature-centric abstracts in Provisioning Hosts (#4958)
theforeman/foreman-documentation: guides/common/modules/con_boot-loader-management-and-file-structure.adoc

[role="_abstract"]

> {Project} manages and serves boot loader binaries through the *{SmartProxy}*, which provides the network boot services used during automated host provisioning.

Reviewer: ```suggestion
{Project} manages and serves boot loader binaries through {SmartProxy}, which provides the network boot services used during automated host provisioning.
```

> You can use PXE boot for BIOS and UEFI firmware on architectures that use TFTP or HTTP(S).

Reviewer: This sounds misleading because it isn't those architectures utilizing TFTP or HTTP(S) - it's the boot utilizing those protocols IMHO.

Author: I don't think my edit changed the meaning much, so let me get @stejskalleos here for confirmation.

We can juust fit in something like:

```suggestion
You can use PXE boot for BIOS and UEFI firmware on architectures with boot loaders that use TFTP or HTTP(S).
```

Reviewer: How about:
```suggestion
You can boot hosts with BIOS and UEFI firmware and various architectures over PXE using TFTP or HTTP(S).
```

Author: I like that, but let's disambiguate the "using". Did I get it right?

```suggestion
You can boot hosts with BIOS and UEFI firmware and various architectures over PXE that use TFTP or HTTP(S).
```

Reviewer: Perhaps:
```suggestion
You can boot hosts with BIOS and UEFI firmware and various architectures over PXE that can use TFTP or HTTP(S).
```

Author: Just discussed with @stejskalleos. Because only x86_64 is supported, mentioning architectures would be misleading:

```suggestion
You can use PXE boot for BIOS and UEFI firmware.
```
